playbooks/init/base_packages.yml

```diff
@@ -39,6 +39,7 @@
       - "{{ 'python-ipaddress' if ansible_distribution != 'Fedora' else '' }}"
       - libsemanage-python
       - yum-utils
+      - python-docker
```

Comment:
Fedora?  We might need python3-docker here?

Reply:
Fixed